proofs/sectorbuilder/rustsectorbuilder.go

```diff
@@ -30,6 +34,10 @@ var log = logging.Logger("sectorbuilder") // nolint: deadcode
 // be open and accepting data at any time.
 const MaxNumStagedSectors = 1
 
+// MaxTimeToWriteBytesToSink configures the maximum amount of time it should
+// take to copy user piece bytes from the provided Reader to the ByteSink.
+const MaxTimeToWriteBytesToSink = time.Second * 30
```

Comment:
On my laptop, I can write approximately 305MiB/second through a pipe from one process to the other. This doesn't involve any preprocessing, of course. Preprocessing isn't particularly slow - and our current (max) piece size is 256MiB. I think 30 seconds is fine, for now.

Reply:
Sounds good!